src/Card/index.jsx

```diff
@@ -11,27 +11,42 @@ import CardImageCap from './CardImageCap';
 import CardBody from './CardBody';
 import CardStatus from './CardStatus';
 
+export const CARD_VARIANTS = ['light', 'dark', 'muted'];
+
 const Card = React.forwardRef(({
   orientation,
   isLoading,
   className,
   isClickable,
   muted,
+  variant,
   ...props
-}, ref) => (
-  <CardContextProvider orientation={orientation} isLoading={isLoading}>
-    <BaseCard
-      {...props}
-      className={classNames(className, 'pgn__card', {
-        horizontal: orientation === 'horizontal',
-        clickable: isClickable,
-        'is-muted': muted,
-      })}
-      ref={ref}
-      tabIndex={isClickable ? '0' : '-1'}
-    />
-  </CardContextProvider>
-));
+}, ref) => {
+  if (muted) {
+    // eslint-disable-next-line no-console
+    console.error('Prop muted is deprecated, please, use variant="muted" instead');
```

Comment:
[curious] Could/should we use `withDeprecatedProps` to mark this as a deprecated prop (see other usages of it in the repo)?

Reply:
yeah, we probably should, updated